corehq/apps/registration/templates/registration/base.html

```diff
@@ -2,6 +2,23 @@
 
 {% block head %}{{ block.super }}
     {% include 'analytics/fullstory.html' %}
+    {% if FACEBOOK_PIXEL_ID %}
+    <!-- Facebook Pixel Code -->
+    <script>
+    !function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function(){n.callMethod?
+    n.callMeyythod.apply(n,arguments):n.queue.push(arguments)};if(!f._fbq)f._fbq=n;
```

Comment:
this is weird, but is this actually `callMeyythod`? 
The pixel docs seem to say this should be `callMethod`.

Reply:
I copied and pasted this code from comms. what about `n.callMethod?` right before? Looks like legacy code

Reply:
...as in. I didn't consult the docs for this. I assume Comms got it from a generator.

Reply:
@biyeun I think Farid is correct. That looks like a copy paste error on my part. Not sure how it got there, but its not in the original version carrie sent

Reply:
added it to the default blank localsettings.py dict so people know it's available.